go.mod

```
@@ -1,10 +1,17 @@
 module github.com/teamwork/utils
```

Should this be a `/v2` as there're breaking changes?
```suggestion
module github.com/teamwork/utils/v2
```